Source/RetrieveAndRankV1/RetrieveAndRank.swift

```diff
@@ -452,22 +482,18 @@ public class RetrieveAndRank {
      
      - parameter solrClusterID: The ID of the cluster to delete this collection from.
      - parameter name: The name of the collection.
-     - parameter writerType: Specifies the QueryResponseWriter to use to process the request. Valid
-            values are declared in the solrconfig.xml file.
      - parameter failure: A function executed if an error occurs.
      - parameter success: A function executed if no error occurs.
      */
     public func deleteSolrCollection(
         solrClusterID: String,
         name: String,
-        writerType: String,
         failure: (NSError -> Void)? = nil,
         success: (Void -> Void)? = nil) {
         
         // construct query parameters
         var queryParameters = [NSURLQueryItem]()
         queryParameters.append(NSURLQueryItem(name: "action", value: "DELETE"))
-        queryParameters.append(NSURLQueryItem(name: "wt", value: writerType))
```

Comment:
Should this be `queryParameters.append(NSURLQueryItem(name: "wt", value: "json"))`?